src/webextension/survey/components/fields/details.jsx

```diff
@@ -0,0 +1,13 @@
+import React, { Component } from 'react'
```

Comment:
Good catch that the JSX files weren't being prettied or linted; I missed that entirely.